### DISCUSSION TITLE
How I change ammo type in RE 7 VR on Valve Index?

### DISCUSSION THREAD

**Discussion by Satan-friend** [praydog/REFramework#962] [General Questions]
I try binding in SteamVR but I see only "RE2_ammotype" but this not work in RE 7 :/

**Comment by devin8or** [comment]
Hold right grip, press the left B button (this is the same button for inventory). You may need to be aiming the weapon/holding it forward to do so, not entirely sure, not playing right now.

I couldn't find anything in the official documentation. I tried many combinations, but never thought to use grip for whatever reason. I managed to get through the base game without switching ammo, but in the Not a Hero DLC, it is required. I Found the answer in this video. https://youtu.be/K_zYXwTNrmo?si=IitOSIEy0T2__2VG
